### PR TITLE
Migrate mobile_app scripts from testeng-ci into this repo

### PR DESCRIPTION
**Pull request by aleffert**
Add mobile app build scripts.
Setup checking python code in the .travis.yml file.

(note that this also includes the changes from aleffert/initial_setup. That will get resolved once that branch gets merged).

### CODE REVIEW

**Comment by aleffert** [comment]
@benpatterson please review. This is broken out in two commits. The first is just moving stuff. The second one connects it to CI

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,17 @@
 sudo: false
 language: groovy
```
Comment (benpatterson) [inline .travis.yml]: I think you can refactor this according to the below pattern. Have you tried that, perchance?
```
language:
  - python
  - groovy
```

Comment (aleffert) [inline .travis.yml]: Oh, I didn't know you could do that - I thought they had different images per language. But I'll give it a shot.

Comment (aleffert) [inline .travis.yml]: For those following along at home, that doesn't work. It just picks the first item.

Comment (benpatterson) [inline .travis.yml]: As discussed offline: bummer. but thanks for looking.

**Comment by benpatterson** [comment]
@aleffert I compared the files using `git diff --no-index` on the old repo, and only see the change from relative imports.

👍

**Pull request merged**